mautrix_telegram/portal/telegram.py

```diff
@@ -65,6 +66,8 @@
 
 
 class PortalTelegram(BasePortal, ABC):
+    latest_event_timestamp: float = time()
```

Comment:
This default seems to indicate there has been an event at some point. `0` would make more sense as a default?

```suggestion
    latest_event_timestamp: float = 0.0
```

Reply:
I picked `time()` so the bridge starts `ready` and gets killed on startup.

Reply:
The timeout now only applies once the first Telegram event has been received.
Until then the timestamp is undefined and does not affect `AppService.live`.